rts/Rendering/QTPFSPathDrawer.cpp

```diff
@@ -38,9 +38,10 @@ static constexpr unsigned char LINK_COLOR[4] = {1 * 255, 0 * 255, 1 * 255, 1 * 1
 static constexpr unsigned char PATH_COLOR[4] = {0 * 255, 0 * 255, 1 * 255, 1 * 255};
 static constexpr unsigned char BAD_PATH_COLOR[4] = {1 * 255, 0 * 255, 0 * 255, 1 * 255};
 static constexpr unsigned char INCOMPLETE_PATH_COLOR[4] = {1 * 255, 1 * 255, 0 * 255, 1 * 128};
-static constexpr unsigned char NODE_COLORS[3][4] = {
+static constexpr unsigned char NODE_COLORS[][4] = {
 	{1 * 255, 0 * 255, 0 * 255, 1 * 255}, // red --> blocked
 	{0 * 255, 1 * 255, 0 * 255, 1 * 255}, // green --> passable
+	{1 * 255, 1 * 127, 0 * 255, 1 * 255}, // orange --> exit only
```

Comment:
`1 * x` and `0 * x` look confusing. They could make some sense if it was always `fraction * 255` but that isn't the case. Consider rebasing onto #1468